rdrf/rdrf/templatetags/has_subscribable_email.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+from rdrf.models.definition.models import EmailNotification
+
+register = template.Library()
+
+
+# check whether the registry the user belongs to has subscribable email notifications.
+@register.filter()
+def has_subscribable_email(user):
+    if user.num_registries == 0:
```

Comment:
This is just querying the same `Registry` model table twice (first with `user.num_registries` and again with `user.registry.all()`)

And it's redundant because the function will return `False` anyway if the iterator in the loop is empty.